include/deal.II/fe/mapping_fe_field.h

```diff
@@ -582,30 +582,35 @@ class MappingFEField : public Mapping<dim,spacedim>
 
 private:
   /**
-   * Update internal degrees of
-   * freedom. */
+   * Update internal degrees of freedom.
+   */
   void update_internal_dofs(const typename Triangulation<dim,spacedim>::cell_iterator &cell) const;
 
+  /**
+   * Typedef holding the local cell_dofs.
+   */
+  typedef
+  Threads::ThreadLocalStorage <std::vector<double> > local_dofs_t;
```

Comment:
Can you rename this to `local_dof_values_t` or `LocalDoFValues` to make it clear that it's the *values*, not the *indices*?